int16 motor1_ecd
int16 motor1_speed_rpm
int16 motor_temp